spec/unit/block_score/person_spec.rb

```diff
@@ -1,135 +1,114 @@
-      end
+    before do
+      create(:person, name_first: 'John', name_last: uniq_token_one)
+      create(:person, name_first: 'John', name_last: uniq_token_two)
     end
 
-    describe '#invalid?' do
-      context 'valid person' do
-        subject(:person) { create(:valid_person) }
-
-        it { is_expected.to be_persisted }
-        it { is_expected.not_to be_invalid }
-        its(:class) { should be BlockScore::Person }
-      end
+    it { is_expected.not_to be_empty }
+    it { expect(people[0].name_last).to eql uniq_token_two }
+    it { expect(people[1].name_last).to eql uniq_token_one }
+  end
 
-      context 'invalid person' do
-        subject(:person) { create(:invalid_person) }
+  describe '#valid?' do
+    context 'valid person' do
+      subject(:person) { create(:valid_person) }
 
-        it { is_expected.to be_persisted }
-        it { is_expected.to be_invalid }
-        its(:class) { should be BlockScore::Person }
-      end
+      it { is_expected.to be_persisted }
+      it { is_expected.to be_valid }
+      it { is_expected.to be_an_instance_of described_class }
     end
 
-    describe '#refresh' do
-      subject(:person) { create(:person, name_first: 'John') }
-      before do
-        person.name_first = 'Mike'
-        person.refresh
-      end
+    context 'invalid person' do
+      subject(:person) { create(:invalid_person) }
 
-      it { expect(person.name_first).to eq 'John' }
+      it { is_expected.to be_persisted }
+      it { is_expected.not_to be_valid }
+      it { is_expected.to be_an_instance_of described_class }
     end
+  end
 
-    describe '#inspect' do
-      subject(:person_inspection) { create(:person).inspect }
-
-      its(:class) { should be(String) }
-      it { is_expected.to match(/^#<BlockScore::Person:0x/) }
-    end
+  describe '#invalid?' do
+    context 'valid person' do
+      subject(:person) { create(:valid_person) }
 
-    describe '#delete' do
-      subject(:person) { create(:person) }
-      it { expect { person.delete }.to raise_error NoMethodError }
+      it { is_expected.to be_persisted }
+      it { is_expected.not_to be_invalid }
+      it { is_expected.to be_an_instance_of described_class }
     end
 
-    describe '#save' do
-      subject(:person) { build(:person) }
-      before { person.save }
+    context 'invalid person' do
+      subject(:person) { create(:invalid_person) }
 
       it { is_expected.to be_persisted }
-      its(:class) { should be BlockScore::Person }
+      it { is_expected.to be_invalid }
+      it { is_expected.to be_an_instance_of described_class }
     end
   end
+
+  describe '#delete' do
+    subject(:person) { create(:person) }
+    it { expect { person.delete }.to raise_error NoMethodError }
+  end
+
+  describe '#save' do
+    subject(:person) { build(:person) }
+    before { person.save }
+
+    it { is_expected.to be_persisted }
+    it { is_expected.to be_an_instance_of described_class }
```

Comment:
I'm curious what the purpose is behind this kind of test? I mean, doesn't the factory explicitly return this specific type of class? Doesn't that make this a test more about the factory than the `#save` method? Even if we wanted to, in ruby the `#save` method can't change an instance type so this doesn't seem necessary.

However, I don't see any spec for the method's return value, which I think is an important detail and mutant most certainly will flag the lack of return value tests.